app/assets/javascripts/direct_uploads.js

```diff
@@ -29,8 +29,14 @@ addEventListener("direct-upload:error", event => {
   event.preventDefault()
   const { id, error } = event.detail
   const element = document.getElementById(`direct-upload-${id}`)
+  Rollbar.critical("ActiveStorage Direct Upload Failed.", event);
   element.classList.add("direct-upload--error")
   element.setAttribute("title", error)
+  element.insertAdjacentHTML("afterend", `
```

Comment:
@frrrances is this going to be difficult for you to maintain if you want to change the UX? Do we need to document this somewhere?